src/weatherreport/src/weatherreport_check_search.erl

```diff
@@ -54,7 +69,11 @@ check(_Opts) ->
     end.
 
 -spec format(term()) -> {io:format(), [term()]}.
+format(clouseau_not_configured) ->
+    {"Clouseau service is not configured", []};
 format({clouseau_ok, SearchNode}) ->
     {"Local search node at ~w responding ok", [SearchNode]};
+format({clouseau_node, SearchNode}) ->
+    {"Search node name ~s is not recognised", [SearchNode]};
```

Comment:
"recognised" is the UK spelling.  I think the US spelling ("recognized") is used in this project, but I may be wrong here.